### PR TITLE
fix(code-examples): satisfy ESLint rules for action button and AG Grid code examples

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: apps/code-examples/src/main.ts [hunk 1]
```diff
@@ -10,4 +10,7 @@ if (environment.production) {
 
 platformBrowserDynamic()
   .bootstrapModule(AppModule)
-  .catch((err) => console.error(err));
+  .catch((err) => {
+    // eslint-disable-next-line no-console
+    console.error(err);
+  });
```
Comment (johnhwhite) [inline apps/code-examples/src/main.ts:16]: Is this still needed?

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2423/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2423/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2423/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2423/playground/)

**Review by johnhwhite — CHANGES_REQUESTED** [review]
A couple of the data manager examples aren't loading data.

**Review by johnhwhite — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/2425

**Pull request merged**